src/Grids/exterior_node.jl

```diff
@@ -0,0 +1,85 @@
+
+const c = Center()
+const f = Face()
+
+const XBoundedGrid = AbstractGrid{<:Any, <:Bounded}
+const YBoundedGrid = AbstractGrid{<:Any, <:Any, <:Bounded}
+const ZBoundedGrid = AbstractGrid{<:Any, <:Any, <:Any, <:Bounded}
+
+const XYBoundedGrid = AbstractGrid{<:Any, <:Bounded, <:Bounded}
+const XZBoundedGrid = AbstractGrid{<:Any, <:Bounded, <:Any, <:Bounded}
+const YZBoundedGrid = AbstractGrid{<:Any, <:Any, <:Bounded, <:Bounded}
+
+const XYZBoundedGrid =  AbstractGrid{<:Any, <:Bounded, <:Bounded, <:Bounded}
+
+#####
+##### Exterior node and peripheral node
+#####
+
+"""
+    exterior_cell(i, j, k, grid)
+
+Return `true` when the tracer cell is outside the boundaries of the domain.
+Return `false` either when the tracer cell at index `i, j, k` is "wet".
```

Comment:
Is there something missing in this sentence?
Also, can we clarify what outside the boundaries of the domain mean, please? (I understand this as being halo regions or inside immersed solid, but not sure if that's exactly the case)

Reply:
The domain is divided into "wet" cells (where we evolve prognostic fields) and "dry" cells. There are dry cells even without the immersed grid.

I think the docstring should be short, but we can include a more involved introduction to the concept of halo regions + immersed cells versus "wet" prognosed cells somewhere and reference it here.

Reply:
I wonder if

```
inactive_node
```

and 

```
inactive_cell
```

are better names than "exterior". What do you think @tomchor @simone-silvestri ?

I'll keep "peripheral".

Reply:
I think inactive cell is more intuitive :+1: